students/kristoffer_jonson/lesson3/list_lab.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+def series_1():
+    """
+    Return list of fruit that has been added to by prompt
+    """
+
+    fruit_list = ['Apples','Pears','Oranges','Peaches']
+    print(fruit_list)
+
+    fruit_to_add = input("What additional fruit sbould be added?")
+    fruit_list.append(fruit_to_add)
+    print(fruit_list)
+
+    fruit_number = input("Request fruit to be shown based on position:")
+    print("You have requested to see fruit {}: {}".format(int(fruit_number), fruit_list[int(fruit_number)-1]))
+
+    #Add another fruit to the beginning of the list using plus and display the list.
+    additional_fruit_to_add = ['Melon']
+    fruit_list = additional_fruit_to_add + fruit_list
+    print(fruit_list)
+
+    #Add another fruit to the beginning of the list using insert() and display the list.
+    additional_fruit_to_add = 'Strawberries'
+    fruit_list.insert(0,additional_fruit_to_add)
+    print(fruit_list)
+
+    fruit_with_p = []
+    for fruit in  fruit_list:
+        if 'p' in fruit.lower():
```

Comment:
per instruction you need to check if fruit starts with "P" -- you're testing if the word contains "P"

you can either use startswith() function or check 0th element.